codecparsers/vc1Parser.cpp

```diff
@@ -232,87 +260,93 @@ namespace VC1 {
     {
         uint32_t i;
         BitReader bitReader(data, size);
```

Comment:
I mean you can change the `BitReader bitReader` to `BitReader br` and use br in your macro. This good since you do not need convert to point.